RFS/build.gradle

```diff
@@ -84,6 +85,9 @@ dependencies {
     testFixturesImplementation group: 'org.testcontainers', name: 'testcontainers'
 
     testFixturesImplementation group: 'org.hamcrest', name: 'hamcrest'
+
+    testImplementation 'org.openjdk.jmh:jmh-core:1.36'
+    testImplementation 'org.openjdk.jmh:jmh-generator-annprocess:1.36'
```

Comment:
Is there a reason not to promote these package versions to the top level gradle dependency definition, like the rest of our dependencies?

Reply:
Good call out, I'll move them up there.